src/Symfony/Component/Serializer/Tests/Fixtures/GroupDummySnakeCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
The license header is missing

Reply:
Yes sorry for that, i begin i try to dont forget next time ;) . thanks

Reply:
No proble :)